config/locales/en.yml

```diff
@@ -72,7 +72,10 @@ en:
     option_add: 'Add option'
     option_remove: 'Delete...'
   branches:
+    add_branch: 'Add another branch'
     after: After "%{page_title}"
+    expression: 'If'
+    expression: 'And'
```

Comment:
`t('branches.expression')` will only ever return `If`?

Reply:
currently neither of these are used

Reply:
Weird. The code is rendering output as though that text is being used but, when I change the text in the actual en.yml file, it's clear that's not actually the case. Not hard to see why when taking a closer look at code used, but still... weird.

Have corrected but going to delete this PR and raise another because I've stuffed up the history.